lux/test/unit/lux/lenses/etherscan/accounts/balance_history_lens_test.exs

```diff
@@ -0,0 +1,177 @@
+defmodule Lux.Lenses.Etherscan.BalanceHistoryLensTest do
+  use UnitAPICase, async: false
+
+  alias Lux.Lenses.Etherscan.BalanceHistoryLens
+
+  setup do
+    # Set up test API key in the configuration
+    Application.put_env(:lux, :api_keys, [
```

Comment:
Why do we need to modify this? This value is global for all tests... so you might endup with unexpected results.